### PR TITLE
fix: update multiformats and ipfs client

### PR DESCRIPTION
**Pull request by v-stickykeys**
### Motivation

Tests have been failing because of an update multiaddr used in our dependencies. Re: #59 

### Changes

- Bumped @ceramicnetwork rc versions
- Removed multiformats
- Bumped ipfs-http-client from 49 to 50

### Notes

Tested locally and not seeing the error

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,24 +32,25 @@
...
     "dag-jose": "^0.3.0",
     "dids": "^2.1.0",
-    "ipfs-http-client": "~49.0.4",
+    "ipfs-http-client": "^50.1.2",
     "jest": "^26.6.3",
     "jest-resolver-enhanced": "^1.0.0",
     "key-did-provider-ed25519": "^1.1.0",
     "key-did-resolver": "^1.1.2-rc2",
-    "multiformats": "3.0.3",
+    "multiformats": "^4.6.3",
```
Comment (oed) [inline package.json:53]: Is `multiformats` really needed now that we have `blockcodec-to-ipld-format`?

Comment (v-stickykeys) [reply package.json:53]: oh good point. will remove it

File: package.json [hunk 2]
```diff
@@ -32,24 +32,24 @@
...
-    "@ceramicnetwork/common": "^1.0.0-rc.6",
-    "@ceramicnetwork/core": "^1.0.0-rc.12",
-    "@ceramicnetwork/streamid": "^1.0.0-rc.3",
-    "@ceramicnetwork/stream-tile": "^1.0.0-rc.6",
-    "@ceramicnetwork/http-client": "^1.0.0-rc.6",
-    "@ceramicnetwork/ipfs-daemon": "^1.0.0-rc.9",
+    "@ceramicnetwork/cli": "^1.1.0-rc",
+    "@ceramicnetwork/common": "^1.1.0-rc",
+    "@ceramicnetwork/core": "^1.2.0-rc",
+    "@ceramicnetwork/http-client": "^1.0.7-rc",
```
Comment (stbrody) [inline package.json:38]: can (need to?) drop the `-rc` from the end

File: package.json [hunk 3]
```diff
@@ -32,24 +32,24 @@
...
+    "@ceramicnetwork/ipfs-daemon": "^1.0.7-rc",
+    "@ceramicnetwork/stream-tile": "^1.0.7-rc",
+    "@ceramicnetwork/streamid": "^1.0.0-rc",
```
Comment (stbrody) [inline package.json:41]: can go to 1.0.2

**Review by stbrody — APPROVED** [review]
small suggestions to the package.json versions, otherwise looks good

File: package.json (hunk 2 above)
Comment (v-stickykeys) [reply package.json:38]: @stbrody drop the rc from all?

File: package.json (hunk 3 above)
Comment (stbrody) [reply package.json:41]: was referring to `streamid` package, though you're right it shouldn't really matter

File: package.json (hunk 2 above)
Comment (stbrody) [reply package.json:38]: yeah.  TBH I didn't even know you *could* just leave the `-rc` suffix without specifying a specific RC version.  I wonder what it even means/does?  Will it just take any RC for version 1.0.7?  In any case would be good to drop and require the actual version 1.0.7 or later, not an RC for it.

**Pull request merged**